scripts/build.tokens.ckerc20.mjs

```diff
@@ -89,6 +121,13 @@ const findCkErc20 = async () => {
 	};
 
 	writeFileSync(join(DATA_FOLDER, 'tokens.ckerc20.json'), JSON.stringify(tokens, jsonReplacer, 8));
+
+	for (const [name, { ledgerCanisterId }] of Object.entries({
```

Comment:
NTH: `await Promise.allSettled(Object.entrie().map(...)`

On one hand, that's an opiniated feedback (😅), on the other it might be interesting to continue to generate the logo if just one ledger is not accessible?

Reply:
indeed, it shouldn't happen, but better to put it there